docs/errorhandling.rst

```diff
@@ -57,19 +57,15 @@ The `YOUR_DSN_HERE` value needs to be replaced with the DSN value you get
 from your Sentry installation.
 
 After installation, failures leading to an Internal Server Error are automatically reported to 
-Sentry and from there you can receive error notifications. Sentry also supports 
-capturing custom exceptions::
-    
-    import sentry_sdk
-    
-    try:
-        throwing_function()
-    except Exception as e:
```

Comment:
I removed this because it's not necessary for basic getting started. Also `push_scope` is not necessary, there is already a scope by default.